libjsengine/JSEngine.h

```diff
@@ -0,0 +1,55 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file JSEngine.h
+ * @author Marek Kotewicz <marek@ethdev.com>
+ * @date 2015
+ * Ethereum client.
+ */
+
+#pragma once
+
+namespace dev
+{
+namespace eth
+{
+
+class JSString
```

Comment:
Unfortunately not :/ Mac version of v8 (downloaded from brew) is built with old standard library (`libstdc++`) and the only way to utilize it, is to link it dynamically and not expose standard library types in the interface (it causes conflicts).